backend/rds.ts

```diff
@@ -90,17 +90,21 @@ export async function ensureDatabase() {
 }
 
 async function createDatabase() {
-  await executeStatementInDatabase(null, "CREATE DATABASE " + dbName);
+  await executeStatementInDatabase(
+    null,
+    `CREATE DATABASE ${dbName}
+    CHARACTER SET utf8mb4`
+  );
 
   await executeStatement(`CREATE TABLE Cookie (
     Version BIGINT NOT NULL)`);
   await executeStatement(`CREATE TABLE Client (
-    Id VARCHAR(255) PRIMARY KEY NOT NULL,
+    Id VARCHAR(100) PRIMARY KEY NOT NULL,
```

Comment:
The max size of keys is measured in bytes. By changing the character set, we exceeded it.